terminus-ui/confirmation/src/confirmation-modal.component.scss

```diff
@@ -32,9 +32,28 @@
   }
 
   // Create the triangle above the popup box
-  &::before {
+  &.ts-confirmation-overlay__panel-above::before {
+    @include triangle(bottom, color(pure), spacing(small, 1));
+    left: 50%;
+  }
+
+  &.ts-confirmation-overlay__panel-below::before {
     @include triangle(top, color(pure), spacing(small, 1));
     left: 50%;
     transform: translateX(-50%);
   }
+
+  &.ts-confirmation-overlay__panel-before::before {
+    @include triangle(right, color(pure), spacing(small, 1));
+    right: -12px;
```

Comment:
We can use `right: -#{spacing(small)}` here and on line 55.

The `#{}` forces interpolation of the function before adding the negative prefix. It doesn't come up much, but can be pretty powerful: https://sass-lang.com/documentation/file.SASS_REFERENCE.html#interpolation_

Reply:
THANKS!